dissect/target/plugin.py

```diff
@@ -920,27 +920,28 @@ def find_plugin_functions(target: Target, patterns: str, compatibility: bool = F
                 funcname = pattern
                 namespace = None
 
-            plugindesc = None
-            for index_name, func in functions.items():
+            plugin_descriptions = []
+            for _, func in functions.items():
                 nsmatch = namespace and func["namespace"] == namespace and funcname in func["exports"]
                 fmatch = not namespace and not func["namespace"] and funcname in func["exports"]
                 if nsmatch or fmatch:
-                    plugindesc = func
+                    plugin_descriptions.append(func)
 
-            if plugindesc:
-                loaded_plugin_object = load(plugindesc)
+            for description in plugin_descriptions:
+                loaded_plugin_object = load(description)
                 fobject = inspect.getattr_static(loaded_plugin_object, funcname)
 
                 if compatibility and not loaded_plugin_object(target).is_compatible():
                     continue
 
                 result.append(
                     PluginFunction(
-                        name=f"{plugindesc['module']}.{pattern}",
+                        name=f"{description['module']}.{pattern}",
                         class_object=loaded_plugin_object,
                         method_name=funcname,
                         output_type=getattr(fobject, "__output__", "text"),
-                        plugin_desc=plugindesc,
+                        plugin_desc=description,
                     )
                 )
-    return result
+
+    return list(set(result))
```

Comment:
How would plugins end up multiple times in result? Or did e.g. `windows.services` and `unix.services` get the exact same `PluginFunction` representation?